packages/uix-core/src/cross-realm-object.ts

```diff
@@ -78,7 +78,9 @@ async function setupApiExchange<T>(
         unsubscribe();
         if (!done) {
           done = true;
-          reject(e);
+          if (e) {
+            reject(e);
+          }
```

Comment:
The `destroyed` event fires whenever there's an error, _but also_ when the iframe is closed with no error. We now detect whether an error was passed instead of rejecting in all circumstances.